aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -191,7 +191,7 @@ When deploying apps to servers with [Web Deploy](/iis/publish/using-web-deploy/i
 
 1. Within the new folder, create a *logs* folder to hold ASP.NET Core Module stdout logs when stdout logging is enabled. If the app is deployed with a *logs* folder in the payload, skip this step. For instructions on how to enable MSBuild to create the *logs* folder automatically when the project is built locally, see the [Directory structure](xref:host-and-deploy/directory-structure) topic.
 
-   **Important!** Only use the stdout log to troubleshoot app startup failures. Never use stdout logging for routine app logging. There's no limit on log file size or the number of log files created. For more information on the stdout log, see [Log creation and redirection](xref:host-and-deploy/aspnet-core-module#log-creation-and-redirection). For information on logging in an ASP.NET Core app, see the [Logging](xref:fundamentals/logging/index) topic.
+   **Important!** Only use the stdout log to troubleshoot app startup failures. Never use stdout logging for routine app logging. There's no limit on log file size or the number of log files created. The app pool must have write access to the location where the logs are written. All of the folders on the path to the log location must exist. For more information on the stdout log, see [Log creation and redirection](xref:host-and-deploy/aspnet-core-module#log-creation-and-redirection). For information on logging in an ASP.NET Core app, see the [Logging](xref:fundamentals/logging/index) topic.
```

Comment:
- Convert this to the IMPORTANT widget
- Should any of these "stdout" references by code-fenced for consistency?

Reply:
It wasn't my impression that I've been code fencing them, but these topics go back quite a way in repo history. I'm not surprised to see a bit of this. I'll add a commit to this PR to fix them all everywhere just so we can knock it out here. However, I prefer them *not* to be code fenced. imo code-fencing slows down reading comprehension and should only be used when it's necessary to separate code (and links) from the flow of the text.

@Rick-Anderson Thoughts?

Reply:
Yes ... as I suspected ... there are only six instances where "stdout" is code-fenced across the entire doc set. There are 30 instances where it isn't code fenced. I prefer them *not* code fenced, so I'd like to remove the code fencing from these six instances.

Reply:
Not code-fenced is fine with me. Lack of consistency is the bigger issue to address.